moon-store-mac-os/moon-store-mac-os/Helpers/AlertManager/AlertManager.swift

```diff
@@ -11,7 +11,7 @@ final class AlertPresenter {
     
     private init() { }
     
-    static func showAlert(type: AlertType, alertMessage: String) {
+    static func showAlert(type: AlertType = .info, alertMessage: String) {
```

Comment:
Usually the params with default values should be the last ones

```suggestion
    static func showAlert(alertMessage: String, type: AlertType = .info) {
```

Reply:
In Swift this rule is not normally used, you can add value by default no matter where the parameter is located, here are some screenshots of how Swift has it even in its framework

<img width="960" alt="image" src="https://github.com/user-attachments/assets/529873e2-b005-448b-bb76-d0c978d2e4dc" />
<img width="786" alt="image" src="https://github.com/user-attachments/assets/5a2ee9bf-a914-41fb-9816-e3db628a4b10" />

Reply:
Done! @DerianCordobaPerez 
<img width="457" alt="image" src="https://github.com/user-attachments/assets/c22bb323-49eb-45b3-8ef5-567c9c991279" />